billing/management/commands/update_delinquents.py

```diff
@@ -0,0 +1,74 @@
+import json
+import logging
+from typing import List, Tuple, Dict
+
+from django.core.management.base import BaseCommand
+
+from ...actions.accounts import update_accounts_delinquent_status
+from ...models import Account
+
+logger = logging.getLogger('django.db.backends')
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler())
+
+
+class Command(BaseCommand):
+    """
+    Execution example:
+        > python manage.py update_delinquents
+                           --unpaid-invoices 2
+                           --days-since-last-unpaid 30
+                           --amount-thresholds '{"CHF": 200, "EUR": 100, "NOK": 1000}'
+    """
+    help = (
+        'This command mark accounts as delinquent and vice versa when some criteria '
+        'are accomplished'
+    )
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--unpaid-invoices',
+            type=int,
+            default=None,
+            help='Number of unpaid invoices to consider an user as a delinquent'
+        )
+        parser.add_argument(
+            '--days-since-last-unpaid',
+            type=int,
+            default=None,
+            help=(
+                'Days to take into account since the last unpaid invoice to consider '
+                'an user as a delinquent'
+            )
+        )
+        parser.add_argument(
+            '--amount-thresholds',
+            type=json.loads,
+            default=None,
+            help=(
+                'Balance threshold to consider an user as a delinquent. '
+                'Ex: \'{"CHF": 200, "EUR": 100, "NOK": 150}\''
+            )
+        )
+
+    def handle(self, *args, **options) -> Tuple[Dict[int, List[str]], List[int]]:
+        unpaid_invoices_threshold = options['unpaid_invoices']
```

Comment:
I'd add a dry-run option so we know what's going to be changed. 
I guess this will imply splitting update_accounts_delinquent_status in two:
a) calculate_accounts_that_changed_status (not a great name)
b) apply changes of status

The dry-run option never calls b, but logs what is going to happen.

Reply:
Done with the next methods:
* `get_accounts_which_delinquent_status_has_to_change`
* `swap_delinquent_status`

Reply:
Done